pubsub/topics/main.go

```diff
@@ -145,6 +149,37 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishThatScales(client *pubsub.Client, topic string, n int) {
```

Comment:
Would you mind adding a test?

Reply:
That's a good idea. This is my first piece of Go code. Hungry for comments/feedback. :)

Reply:
I have added a return value for PublishThatScales and wrote a test for it.